package.json

```diff
@@ -10,7 +10,7 @@
   "license": "",
   "description": "Perf Widget front-end and back-end",
   "engines": {
-    "node": "5.x.x"
+    "node": "5.6.0"
```

Comment:
Heroku doesn't recommend having a floating version number on production sites. (Was reading the docs)